handlers/main.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Download and unpack latest syslinux
+  unarchive:
+    remote_src: yes
+    src: https://mirrors.edge.kernel.org/pub/linux/utils/boot/syslinux/Testing/{{ syslinux_latest.stdout }}/syslinux-{{ syslinux_latest.stdout }}-pre1.tar.gz
```

Comment:
Why not download the file to `global_cache_dir`? That way, the server does not need internet access.

Reply:
I opened in issue #5